### ISSUE TITLE
failed to normalize pprof: Key cannot be empty

### ISSUE THREAD

**Issue by guettli** [parca-dev/parca#671]
I try to use parca for the first time.

I followed the instructions for "binary":

```
curl -sL https://github.com/parca-dev/parca/releases/download/v0.8.0/parca_0.8.0_`uname -s`_`uname -m`.tar.gz | tar xvfz -

# Get basic configuration
curl -sL https://raw.githubusercontent.com/parca-dev/parca/main/parca.yaml > parca.yaml

# Run Parca and access the Web UI on port 7070
./parca


level=info name=parca ts=2022-02-24T11:22:52.672807535Z caller=badger.go:100 msg="Set nextTxnTs to 0"
level=info name=parca ts=2022-02-24T11:22:52.673137161Z caller=factory.go:49 msg="loading bucket configuration"
level=info name=parca ts=2022-02-24T11:22:52.673329121Z caller=factory.go:49 msg="loading bucket configuration"
level=info name=parca ts=2022-02-24T11:22:52.673454137Z caller=server.go:92 msg="starting server" addr=:7070

level=error name=parca ts=2022-02-24T11:22:59.934256285Z caller=scrape.go:484 scrape_pool=default target=http://127.0.0.1:7070/debug/pprof/threadcreate msg="WriteRaw failed for scraped profile" err="rpc error: code = Internal desc = failed to normalize pprof: Key cannot be empty"

level=error name=parca ts=2022-02-24T11:23:02.93316081Z caller=scrape.go:484 scrape_pool=default target=http://127.0.0.1:7070/debug/pprof/threadcreate msg="WriteRaw failed for scraped profile" err="rpc error: code = Internal desc = failed to normalize pprof: Key cannot be empty"
```
> failed to normalize pprof: Key cannot be empty"

Should I ignore this error message, or is there something wrong?

My system:

```
Ubuntu 21.10
Linux p15 5.13.0-28-generic #31-Ubuntu SMP Thu Jan 13 17:41:06 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by brancz** [comment]
Could you try latest `main`? I believe this was fixed by #666, and ultimately the reason seems to be #670.

**Comment by guettli** [comment]
@brancz yes, if I use latest main, then there is no such error message.

I leave it up to you, what to do next. For me everything works fine.

**Comment by brancz** [comment]
We are now both ignoring samples that have no stacktrace as well as removed the threadcreate profile from the default configuration and released both of those patches as part of v0.8.1 yesterday. Do you think this is resolved with that?

**Comment by guettli** [comment]
Yes, looks good. I close this issue.
